cimr/processor/process.py

```diff
@@ -340,54 +318,54 @@ def make_composite_id(self):
 
         if set(VAR_COMPONENTS).issubset(set(self.included_header)):
             logging.debug(f' checking {VAR_COMPONENTS} for missing values.')
-            self.summary_data.dropna(
+            self.data.dropna(
                 subset=VAR_COMPONENTS, inplace=True
             )
             logging.debug(f' rows with missing {VAR_COMPONENTS} are dropped.')
 
             logging.debug(f' making variant_id using {VAR_COMPONENTS}.')
-            self.summary_data['chrom'] = self.summary_data['variant_chrom']
+            self.data['chrom'] = self.data['variant_chrom']
             self.check_chrom()
             logging.info(f' verifying variant positions are int values.')
-            self.make_int('variant_pos')
+            self.data = make_int(self.data, 'variant_pos')
```

Comment:
I don't understand why this helper function makes things easier. It looks like it'd be simpler to just do the cast here.

Reply:
I agree (unless `make_int` may become more complex in the future).

Reply:
It will get more complex but I will pull it out then